src/components/GravatarProfile/GravatarProfile.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import md5 from 'md5';
+
+import './GravatarProfile.css';
+
+const GravatarProfile = ({email, gravStyle}) => {
+	let hash = email? md5(email.trim().toLowerCase()):'',
+		opts = gravStyle? '?d='+gravStyle:'';
+	return (
+		<div className='GravatarProfile'>
+			<img src={'https://www.gravatar.com/avatar/' + hash + opts} alt='Gravatar avatar'/>
```

Comment:
alt text isn't required on decorative images. - Can this be an empty string instead?